### ISSUE TITLE
<-- + note = the location of the Comment is INCORRECT

### ISSUE THREAD

**Issue by devwitalik** [plantuml/plantuml#1679]
**Steps to reproduce the behavior:**
```plantuml
@startuml
Test --> Test: Text
note right: the location of the Comment is correct

Test --> Test: Text
note left: the location of the Comment is correct

Test <-- Test: Text
note right: the location of the Comment is INCORRECT

Test <-- Test: Text
note left: the location of the Comment is INCORRECT
@enduml
```
http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuGf9BIvHqDMrKm0nh81aHGbNNdv9ga9HPde6KA0a8rKX9pyviIGpFqyXFmtCTyxFpKtDAr78B5P8pYygIaqkuS9jJavg6gd6sEZgakaSJpzd_w0WL-SGX4daEGXXa4DgNWeme000

**Screenshots**

![IMG](http://www.plantuml.com/plantuml/png/SoWkIImgAStDuGf9BIvHqDMrKm0nh81aHGbNNdv9ga9HPde6KA0a8rKX9pyviIGpFqyXFmtCTyxFpKtDAr78B5P8pYygIaqkuS9jJavg6gd6sEZgakaSJpzd_w0WL-SGX4daEGXXa4DgNWeme000)

**Comment by jimnelson372** [comment]
Hello.  When working on some related fixes (for #1680 and #839) I saw that comments to the left or right of left directed self messages were off.   Here is how it looks with my current changes.  Please confirm this is what you would expect.  I only changed your script from INCORRECT to [now correct?], otherwise it's the same script you shared:

<img width="682" alt="image" src="https://github.com/plantuml/plantuml/assets/66284321/b0a613de-614e-4e08-a963-b5e9481d6fec">

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_ 👀 
This is now fixed on [V1.2024.5](https://github.com/plantuml/plantuml/releases/tag/v1.2024.5).
- Thanks @devwitalik for the report.
- Thanks PlantUML team and @jimnelson372 for your work.

Regards.
